projects/clas12/targets/geometry.py

```diff
@@ -504,13 +461,1169 @@ def build_geometry_pol_targ(configuration):
+		r_out = 5.0
+		half_length = 0.015 # (0.030 mm thick)
+		gvolume = GVolume("AlTargetFoilDownstream")
+		gvolume.mother = "targetCell"
+		gvolume.description ="Aluminum Downstream Foil"
+		gvolume.color =  "aaaaaa"
+		gvolume.material = "G4_Al"
+		gvolume.setPosition(0,0,z_center)
+		_make_full_tube(gvolume, r_in, r_out, half_length)
+		return gvolume
+
+	def build_flux_detector():
+		# flux detector downstream of the scatter_ing chamber
+		z_center = 300.0   
+		r_out = 45.0
+		r_in = 0.0
+		half_length = 0.015 # (0.030 mm thick)
+		gvolume = GVolume("testFlux")
+		gvolume.mother = "root"
+		gvolume.description = "Flux detector"
+		gvolume.color = "009900"
+		gvolume.setPosition(0,0,z_center)
+		_make_full_tube(gvolume, r_in, r_out, half_length)
+		gvolume.material = "G4_AIR"
+		# @mariakzurek: sensitivity, hit_type, identifiers are not part of the current gemc3 API
+		# gvolume.sensivity = "flux"
+		# gvolume.hit_type = "flux"
+		# gvolume.identifiers = "id manual 1"
+		return gvolume
+
+	for builder in [
+		build_helium_bag,
+		build_pb_125_microns,
+		build_upstream_foil_al,
+		build_downstream_foil_al,
+		build_flux_detector,
+	]:
+		volume = builder()
+		volume.publish(configuration)
+
+
+def build_geometry_nd3(configuration):
+
+	variation = configuration.variation
+
+	def build_vacuum_container():
+		r_in = 0.0
+		r_out= 44.0
+		half_length = 50.0  # half length
+		gvolume = GVolume("scatter_ingChamberVacuum")
+		gvolume.description = f'clas12 scatter_ing chamber vacuum rohacell container for {variation} target'
+		gvolume.color = "aaaaaa4"
+		_make_full_tube(gvolume, r_in, r_out, half_length)
+		gvolume.material  = "G4_Galactic"
+		return gvolume
+
+	def build_rohacell():
+		r_in = 0.0
+		r_out = 43.0
+		half_length = 48.0  # half lSength
+		gvolume = GVolume("scatter_ingChamber")
+		gvolume.mother = "scatter_ingChamberVacuum"
+		gvolume.description = f'clas12 rohacell scatter_ing chamber for {variation} target'
+		gvolume.color = "ee3344"
+		_make_full_tube(gvolume, r_in, r_out, half_length)
+		gvolume.material  = "rohacell"
+		return gvolume
+
+	def build_vacuum_container_for_plastic_cell():
+		r_in = 0.0
+		r_out = 40.0
+		half_length = 45.0  # half length
+		gvolume = GVolume("plasticCellVacuum")
+		gvolume.mother = "scatter_ingChamber"
+		gvolume.description = f'clas12 rohacell vacuum aluminum container chamber for {variation} target'
+		gvolume.color = "aaaaaa4"
+		_make_full_tube(gvolume, r_in, r_out, half_length)
+		gvolume.material  = "G4_Galactic"
+		return gvolume
+
+	def build_helium_cylinder():
+		r_in = 0.0
+		r_out = 12.62
+		half_length = 25.10  # half length
+		gvolume = GVolume("HeliumCell")
+		gvolume.mother = "plasticCellVacuum"
+		gvolume.description = f'Helium volume for {variation} target'
+		gvolume.color = "aaaaaa3"
+		_make_full_tube(gvolume, r_in, r_out, half_length)
+		#gvolume.material  = "G4_He"
+		gvolume.material  = "G4_Galactic" #temporaly replacing with vacuum. @mariakzurek: What should we do here?
```

Comment:
@maureeungaro Please check this comment

Reply:
G4_He is part of the [G4 material database](https://geant4-userdoc.web.cern.ch/UsersGuides/ForApplicationDeveloper/html/Appendix/materialNames.html) so it's ok to use it here. 

There are now other materials defined in gemc3, in glibrary/g4system/g4world.cc that apply to the targets:

These are the names: 

// predefined isotopes / elements / materials
#define HYDROGEN_ELEMENT      "Hydrogen"
#define DEUTERON_ISOTOPE      "Deuteron"
#define DEUTERIUM_ELEMENT     "Deuterium"
#define HGAS_MATERIAL         "HydrogenGas"
#define DEUTERIUMGAS_MATERIAL "DeuteriumGas"
#define LD2_MATERIAL          "LD2"
#define NITRO_ELEMENT         "Nitrogen"
#define ND3_MATERIAL          "ND3"
#define HELION_ISOTOPE        "Helion"
#define HELIUM3_ELEMENT       "Helium3"
#define HELIUM3GAS_MATERIAL   "Helium3Gas"
#define TRITON_ISOTOPE        "Triton"
#define TRITIUM_ELEMENT       "Tritium"
#define TRITIUMGAS_MATERIAL   "H3Gas"

Reply:
I will change the volume material to `G4_He` then.